src/System.Net.Security/tests/UnitTests/System/Security/Authentication/ExtendedProtection/ExtendedProtectionPolicyTest.cs

```diff
@@ -44,6 +48,21 @@ public void Constructor_CollectionParam()
             Assert.Equal(expectedServiceNameCollection, extendedProtectionPolicy.CustomServiceNames);
         }
 
+        [Fact]
+        public void Constructor_PolicyEnforcement_MembersAreSet()
+        {
+            // Arrange
```

Comment:
It's fine to organize tests in this general pattern, but we don't include comments naming the sections.